.env

```diff
@@ -0,0 +1,8 @@
+# Environment variables declared in this file are automatically made available to Prisma.
```

Comment:
please never commit env files and secrets to a git repo. you can add this to .gitignore